_posts/2021-07-01-release-1.13.2.md

```diff
@@ -0,0 +1,265 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22796'>FLINK-22796</a>] -         Update mem_setup_tm documentation
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22814'>FLINK-22814</a>] -         New sources are not defining/exposing checkpointStartDelayNanos metric
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22815'>FLINK-22815</a>] -         Disable unaligned checkpoints for broadcast partitioning
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22820'>FLINK-22820</a>] -         Stopping Yarn session cluster will cause fatal error
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22833'>FLINK-22833</a>] -         Source tasks (both old and new) are not reporting checkpointStartDelay via CheckpointMetrics
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22856'>FLINK-22856</a>] -         Move our Azure pipelines away from Ubuntu 16.04 by September
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22863'>FLINK-22863</a>] -         ArrayIndexOutOfBoundsException may happen when building rescale edges
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22886'>FLINK-22886</a>] -         Thread leak in RocksDBStateUploader
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22890'>FLINK-22890</a>] -         Few tests fail in HiveTableSinkITCase
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22894'>FLINK-22894</a>] -         Window Top-N should allow n=1
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22908'>FLINK-22908</a>] -         FileExecutionGraphInfoStoreTest.testPutSuspendedJobOnClusterShutdown should wait until job is running
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22927'>FLINK-22927</a>] -         Exception on JobClient.get_job_status().result()
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22932'>FLINK-22932</a>] -         RocksDBStateBackendWindowITCase fails with savepoint timeout
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22945'>FLINK-22945</a>] -         StackOverflowException can happen when a large scale job is CANCELING/FAILING
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22946'>FLINK-22946</a>] -         Network buffer deadlock introduced by unaligned checkpoint
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22948'>FLINK-22948</a>] -         Scala example for toDataStream does not compile
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22952'>FLINK-22952</a>] -         docs_404_check fail on azure due to ruby version not available
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22955'>FLINK-22955</a>] -         lookup join filter push down result to mismatch function signature
```

Comment:
[FLINK-22955] is Unresolved, should we remove it from the 1.13.2 list?
If yes, maybe it's better to double check if there are other unfinished issues in the list?

Reply:
Sure, thanks for your reminder.